### PR TITLE
[BE] Add selected custom ops to CI

### PR DESCRIPTION
**Pull request by kimishpatel**
Summary:
Earlier custom sdpa and kv cache werent being tested in OSS CI. This diff changes that.

Tests
CI

ghstack-source-id: 35586450183e53b22971190f939bd6e362b3859a
Pull Request resolved: https://github.com/pytorch/executorch/pull/11743

### Summary
[PLEASE REMOVE] See [CONTRIBUTING.md's Pull Requests](https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#pull-requests) for ExecuTorch PR guidelines.

[PLEASE REMOVE] If this PR closes an issue, please add a `Fixes #<issue-id>` line.

[PLEASE REMOVE] If this PR introduces a fix or feature that should be the upcoming release notes, please add a "Release notes: <area>" label. For a list of available release notes labels, check out [CONTRIBUTING.md's Pull Requests](https://github.com/pytorch/executorch/blob/main/CONTRIBUTING.md#pull-requests).

### Test plan
[PLEASE REMOVE] How did you test this PR? Please write down any manual commands you used and note down tests that you have written if applicable.

### CODE REVIEW

File: extension/llm/custom_ops/test_sdpa_with_kv_cache.py [hunk 1]
```diff
@@ -604,6 +608,9 @@ def test_sdpa_with_cache_seq_len_llava_example(self):
             n_heads_kv, n_heads_q, head_dim, max_seq_len, seq_len, next_iter_seq_len
         )
 
+    @unittest.skipIf(
+        not is_fbcode(), "in OSS error is too large 0.0004 for some reason"
```
Comment (mergennachin) [inline extension/llm/custom_ops/test_sdpa_with_kv_cache.py:612]: Let's enable in OSS. Otherwise we will never enable in OSS.

We can finetune atol and rtol during comparison.

Comment (kimishpatel) [reply extension/llm/custom_ops/test_sdpa_with_kv_cache.py:612]: I am little reluctant on this because on one hand yes we can make it pass by tuning this knob, but on the other hand if there is an issue we wont catch it. My preference would be to have this test at least running internally without compromise. I dont know the reason as to why it requires low atol

File: extension/llm/custom_ops/test_quantized_sdpa.py [hunk 2]
```diff
@@ -386,6 +391,9 @@ def _test_sdpa_common(
         )
         self.assertTrue(torch.allclose(ref_output, op_output, atol=atol))
 
+    @unittest.skipIf(
+        not is_fbcode(), "in OSS error is too large 0.0002 for some reason"
```
Comment (metascroy) [inline extension/llm/custom_ops/test_quantized_sdpa.py:395]: What does ""in OSS error is too large 0.0002 for some reason" mean?

**Review by metascroy — APPROVED** [review]

File: extension/llm/custom_ops/test_quantized_sdpa.py (hunk 2 above)
Comment (kimishpatel) [reply extension/llm/custom_ops/test_quantized_sdpa.py:395]: When I ran this test in OSS error i got was 0.0002. Which is too large for this test to pass. It does pass internally so not sure if this is blas lib issue or mac vs linux. Bug I wanted to enable the tests in oss since we have to coverage otherwise
